components/client.jsx

```diff
@@ -22,9 +22,7 @@ Router.run(routes, Router.HistoryLocation, function (Handler, state) {
   var pathname = state.query.pathname || state.pathname;
   var lang = i18n.isSupportedLanguage(i18n.currentLanguage) ? i18n.currentLanguage : i18n.defaultLang;
   var query = state.query.pathname ? {} : state.query;
-  if(state.params.currency) {
-    values = currencies[state.params.currency];
-  }
+  values = query.currency && currencies[query.currency] ? currencies[query.currency] : currencies['usd'];
```

Comment:
Totally, you leave in the check. It would look something like this:

`var values = query.currency && currencies[query.currency] ? currencies[query.currency] : currencies['usd'];`

Then remote the `var values = {};` initialization at the top.